person/person.go

```diff
@@ -105,3 +105,126 @@ func (p Person) Validate() []response.Error {
 
 	return errs
 }
+
+func (p Person) IndexConfig() map[string]interface{} {
```

Comment:
I agree, but it will add a bit of complexity because currently the service is compiled and deployed as a single executable file. We'd have to deploy it with a set of config files that are read and interpreted at run time as opposed to during compilation, which would affect the way we test it.

I basically treat it as a constant, as it's not going to change often or between environments. Do you mind if we merge it as is and spin off a separate ticket to work out how best to do it and assess the benefit vs effort/complexity etc.?

Reply:
Yes that's fair enough